Assignment7/src/assignment7/DiceTower.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment7;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ *
+ * @author mkavic
+ */
+public class DiceTower {
+//fields
+private int PANELS;
+List <Die> dice;
+private int trayValue;
+private int value;
+
+/**
+ * Constructor
+ */
+public DiceTower(){
+    this.dice = new ArrayList();
+    value = 0;
+    trayValue = 0;
+}
+/**
+ * Constructor
+ * @param dice 
+ */
+public DiceTower (List dice){
+        this.dice = dice;
+        value = 0;
+        trayValue = 0;
+    }
+ 
+/**
+ * dropDice method
+ */
+    public void dropDice(){
+        for (int i = 0; i < dice.size(); i++){
+            for(int p =0; p <PANELS; p++){
+            this.dice.get(i).roll();
+        }
+        trayValue += this.dice.get(i).value();
```

Comment:
this is a partial violation of the Single Responsibility Principle.

This method should just deal with dropping the dice

tray value should handle reading the values of the dice there. This a good idea because it saves you a loop though